### ISSUE TITLE
error in magpy3 test

### ISSUE THREAD

**Issue by duvayu** [WatsonLab/MAGpy#16]
snakemake --use-conda -s MAGpy test
SyntaxError in line 66 of /MAGpy/MAGpy:
Command must be given as string after the shell keyword. (MAGpy, line 66)

**Comment by duvayu** [comment]
making this change with solve the problem.
```
rule diamond_bin_summary:
    input: expand("diamond_report/bin.{sample}.tsv", sample=IDS)
    output: "diamond_bin_report.tsv"
    shell:'''echo -e 'name\tnprots\tnhits\tnfull\tgenus\tngenus\tspecies\tnspecies\tavgpid' >> {output}
    find diamond_report/ -name "bin*.tsv" | xargs -I {{}} cat {{}} >> {output}'''
```

**Comment by mw55309** [comment]
Yeah sorry about this, it all looked fine to me, but replacing tabs with spaces, in a way that doesn't make sense, fixed it for me in Python 3.6
